sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -279,17 +279,23 @@
 
   /**
    * @since 2.11
+   * @deprecated in 6.7. See {@link Server#getPermanentServerId()}
    */
+  @Deprecated
   String ORGANISATION = "sonar.organisation";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)